tests/evm.rs

```diff
@@ -249,7 +249,7 @@ fn ethereum_solidity_tests() {
         })
         .sum();
 
-    assert_eq!(errors, 1003);
+    assert_eq!(errors, 1004);
```

Comment:
This regression is fine; [it's a bug in solc](https://github.com/ethereum/solidity/issues/11573)

Reply:
Needs a comment or exclusion

Reply:
Good point, excluded it instead